src/main/java/bio/terra/service/dataset/flight/ingest/IngestCreateTargetDataSourceStep.java

```diff
@@ -45,6 +45,8 @@ public StepResult doStep(FlightContext context) throws InterruptedException {
 
     AzureStorageAccountResource storageAccountResource =
         resourceService.getOrCreateStorageAccount(dataset, billingProfile, flightId);
+    workingMap.put(IngestMapKeys.STORAGE_ACCOUNT_RESOURCE, storageAccountResource);
```

Comment:
I wonder if this serializes OK?  I've noticed issues then the object contains open api enums.  I guess in our case they're our own enums (with resourcetype and region)

Reply:
See [comment above](https://github.com/DataBiosphere/jade-data-repo/pull/1051#discussion_r704721231) - We've been using this pattern in our other flights and haven't run into any issues yet, but doesn't mean we won't!

Reply:
Ah got it!

Reply:
Seems to serialize OK in tests.

Reply:
Yep, fair enough